pages/index.vue

```diff
@@ -40,6 +40,8 @@ export default Vue.extend({
     };
   },
   async created() {
+    this.$page.setTitle('Home');
```

Comment:
Should this be translated?

Reply:
Indeed, seems like I forgot about it :yum: